crates/re_selection_panel/src/visible_time_range_ui.rs

```diff
@@ -173,10 +176,27 @@ fn query_range_ui(
     let time_type = time_ctrl.timeline().typ();
 
     let mut interacting_with_controls = false;
+    let markdown = format!(
+        "# Visible time range\n
+This feature controls the time range used to display data in the space view.
+
+The settings are inherited from the parent entity or enclosing space view if not overridden.
+
+Visible time range properties are stored separately for each _type_ of timelines. They may differ \
+depending on whether the current timeline is temporal or a sequence. The current settings apply to \
+all _{}_ timelines.
+
+Notes that the data current as of the time range starting time is included.",
+        match time_type {
+            TimeType::Time => "temporal",
+            TimeType::Sequence => "sequence",
+        }
+    );
 
```

Comment:
I see you took this over from the old text but this is no longer true! We are storing time per timeline name since 0.16! Check https://rerun.io/docs/howto/fixed-window-plot#example-syntax

Reply:
right!